tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -305,6 +311,16 @@ public:
    /// Throws an exception if the field is null.
    void AddField(std::unique_ptr<RFieldBase> field);
 
+   /// Register a subfield so it can be read directly from entries belonging to the model. Because registering a
+   /// subfield does not fundamentally change the model, previously created entries will not be invalidated (but also
+   /// not contain the subfields).
```

Comment:
This sounds ambiguous.  One reading is to mis-read the last use of `subfields` (in `but also not contain the subfields`) to be more generic than intended and mis-understand that the information corresponding to the data member (aka the `subfield` or the `target` of the `subfield`) is no longer available in the previous entries. 
Consider:
```suggestion
   /// subfield does not fundamentally change the model, previously created entries will not be invalidated (nor
   /// changed in any way; the subfield create error is explicitly an accessor added to the current entry).
```